examples/README.adoc

```diff
@@ -100,6 +100,8 @@ Number of Examples: 88 (0 deprecated)
 
 | link:cassandra-kubernetes/README.adoc[Cassandra Kubernetes] (cassandra-kubernetes) | Cloud | An example with Camel and Cassandra running on Kubernetes
 
+| link:google/readme.md[Google] (google) | Cloud | Multiple Camel Google examples
```

Comment:
The file `google/readme.md` doesn't exist

Reply:
To remain homogeneous, let's use the same type of doc file which is `README.adoc`

Reply:
this is auto-generated during the maven build - Added new README.adoc file dor the google subdirectory containing information regarding examples in that directory.